quicklogic/README.md

```diff
@@ -108,10 +151,18 @@ To to add a new design to the flow, and use it as a test follow the guide:
 1. Now enter the build directory of your project and run the appropriate target as described:
 
    ```bash
-   cd build/quicklogic/<faimly>/tests/<your_directory_name>
-   make <your_design_name>-<target_board_name>_<stage_name>
+   cd build/quicklogic/<family>/tests/<your_directory_name>
+   make <your_design_name>-<target_board_name>_bit
    ```
 
 ## Known limitations
 
-SymbiFlow support for QuickLogic FPGAs is currently under heavy development. The current support for the qlf_k4n8 family does not support binary bitstream generation. FASM file generation is supported but is still being worked on.
+SymbiFlow support for Quicklogic FPGAs is currently under heavy development. Following, there are a known set of limitations for the supported architectures
+
+### EOS-S3
```

Comment:
Known limitations no longer apply, the whole section can be removed now.

Reply:
Known limitations section is removed